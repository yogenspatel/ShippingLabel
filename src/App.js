import React from 'react';
import Login from './core/components/login';
import './App.css';
import ShippingHeader from './core/components/header';

function App() {
  return (
    <div className="container">
      <div className="row">
        <div className="col">
          <ShippingHeader />
          <Login />
        </div>
      </div>
    </div>
  );
}

export default App;
