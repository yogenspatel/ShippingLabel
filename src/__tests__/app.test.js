/* eslint-disable no-undef */

import React from "react";
import ReactDOM from "react-dom";
import App from "../App";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";


describe("App renders correctly", () => {
  it("has rendered without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
  });
  it("has called snapshot with App component", () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
