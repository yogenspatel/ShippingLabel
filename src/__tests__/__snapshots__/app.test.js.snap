// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly has called snapshot with App component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <_default />
      <LoginParent />
    </div>
  </div>
</div>
`;
